Octokit/Models/Response/AccessToken.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class AccessToken
+    {
+        public AccessToken() { }
+
+        public AccessToken(string token, DateTimeOffset expiresAt)
+        {
+            Token = token;
+            ExpiresAt = expiresAt;
+        }
+
+        /// <summary>
+        /// The access token
+        /// </summary>
+        public string Token { get; protected set; }
+
+        /// <summary>
+        /// The expiration date
+        /// </summary>
+        public DateTimeOffset ExpiresAt { get; protected set; }
+
+        internal string DebuggerDisplay
+        {
+            get
+            {
+                return string.Format(CultureInfo.InvariantCulture, "Token: {0}, ExpiresAt: {1}", Token, ExpiresAt);
```

Comment:
@itaibh - Just a minor simplification, this Format could be removed I think and use instead:

`   
   return $"{nameof(Token)}: {Token}, {nameof(ExpiresAt)}: {ExpiresAt}";
`

Reply:
This is a brand new feature of C# and won't compile on older versions of the compiler. Are you sure you want to switch to this?

Reply:
We have previuosly discussed this and dont have a problem in using the string interpolation `$"blah {var} blah"` however would prefer this to be done across the code base in a consistent way via a Milestone and one/many PR's only making that change... rather than randomly using it in a few places while 100s of existing cases are still doing it "the old way").

Reply:
I agree with Ryan, I was not suggesting we do this piecemeal, I saw some earlier PR that used this string interpolation so I assumed it was established across the project, but if it isn't I agree 100% that this should be a dedicated focused PR that changes this across the board.